backend/pick-git/build.gradle

```
@@ -27,6 +27,12 @@ dependencies {
 
 	asciidoctor 'org.springframework.restdocs:spring-restdocs-asciidoctor:1.2.6.RELEASE'
 
+	compileOnly 'org.projectlombok:lombok:1.18.20'
+	annotationProcessor 'org.projectlombok:lombok:1.18.20'
+
+	testCompileOnly 'org.projectlombok:lombok:1.18.20'
+	testAnnotationProcessor 'org.projectlombok:lombok:1.18.20'
+
 	testImplementation('org.springframework.restdocs:spring-restdocs-mockmvc')
```

```suggestion
	testImplementation 'org.springframework.restdocs:spring-restdocs-mockmvc'
```

테스트 패키지 구조랑 관련 없는 부분이긴 한데, 여기 소괄호 삭제하고 머지해주면 좋을 거 같아요 !